CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Changelog for NeoFS Node
 - Flush write-cache when moving shard to DEGRADED mode (#1825)
 - Make `morph.cache_ttl` default value equal to morph block time (#1846)
 - Policer marks nodes under maintenance as OK without requests (#1680)
+- Reading `object` commands of NeoFS CLI don't open remote sessions (#1865) 
```

Comment:
```suggestion
- Reading `object` commands of NeoFS CLI doesn't open remote sessions (#1865) 
```

Reply:
I wanted `commands` to be subject, I'll try to fix grammar

Reply:
oh that's my fault, i read that as "reading ... don't", dont remember why

you were correct